api/models.py

```diff
@@ -66,6 +67,18 @@ class Config:
         }
 
 
+class PageModel(LimitOffsetPage[TypeVar("T")]):
+    """Model for pagination
+
+    This model is required to serialize paginated model data response"""
+
+    class Config:
+        """Configuration attributes for PageNode"""
+        json_encoders = {
+            ObjectId: str,
+        }
```

Comment:
I'd think of moving this model to different module. `models.py` seems to be a place for data models. These are connected with the application logic, while `PageModel` is rather connected with data presentation.

Reply:
Makes sense. Added `paginator_models.py`